modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -577,7 +577,7 @@ export const console: ConsoleNS = {
                             }
                         }
                     },
-                    placeholder: "Search by application name or clientId"
+                    placeholder: "Search applications by name or clientId or issuer"
```

Comment:
Is the placeholder fit into the space?

Reply:
On line 434 in apps/console/src/features/applications/pages/applications.tsx file, I added a custom style to make the input box longer. So this placeholder can fit to that.